Source/ARTPush.m

```diff
@@ -54,6 +55,16 @@ - (void)didFailToRegisterForRemoteNotificationsWithError:(NSError *)error {
     [_logger error:@"ARTPush: device token not received (%@)", [error localizedDescription]];
 }
 
++ (ARTPushActivationStateMachine *)activationMachine {
+    static dispatch_once_t once;
+    static id activationMachineInstance;
+    dispatch_once(&once, ^{
+        // Error: ARTPush.m:62:81: Instance variable '_httpExecutor' accessed in class method
+        //activationMachineInstance = [[ARTPushActivationStateMachine alloc] init:_httpExecutor];
+    });
```

Comment:
@tcard I can do requests without the auth token or key, right? I don't really need the Rest instance. So, I can create the singleton with no dependencies.